### ISSUE TITLE
Change default log level to `log`

### ISSUE THREAD

**Issue by benoit74** [openzim/mwoffliner#2121]
Currently, scraper default log level is `error` meaning mostly no log is produced unless an error occurs.

Probably biased by my own expectations, I feel like this is not a "sensible" default, I would expect most user to be presented sufficient logs by default, to know what the tool is currently doing and potential issues encountered (warnings). 

Should the user want more or less logs, he will then find the `--verbose` setting and customize this log level.

I hence suggest that we change the default log level to `log` (meaning we will log by default `log`, `warn` and `error` messages).

At least this is not the first time that I come back to this tool, and every-time I struggle to find again the proper way to print more logs on my console.

WDYT?

**Comment by audiodude** [comment]
I agree wholeheartedly that the logging needs to be updated. If we change the level to `log`, I would also want to do an audit of existing messages to keep the logspam down.

Possibly related, I've noticed that there are at least a few places where the code essentially swallows exceptions, by catching the exception, logging a simple message without a stack trace, and then either rethrowing or failing in another way. Fixing these would also help with general debugging when we get errors in zimfarm, because we would be able to see the stack trace in the immediate logs without having to re-run the job manually.

**Comment by benoit74** [comment]
I've changed log level to `log` in the Zimfarm since few weeks and it looks fine. We will always be able to refine it should we detect something too verbose.
